index.js

```diff
@@ -27,6 +27,10 @@ function makeModule(Promise) {
     return this.toPromise().then(onFulfilled, onRejected);
   }
 
+  function _catch(onRejected) {
+    return this.toPromise().catch(onRejected);
+  }
```

Comment:
Can you pass all arguments through? `_catch() { return this.toPromise().catch(arguments) }`

Bluebird in particular supports a more elegant form of catch that takes multiple arguments.